### PR TITLE
game compatibility: add Kingdom Come: Deliverance & Tomb Raider GOTY

### PR DESCRIPTION
**Pull request by quantenzitrone**
I'm not sure what or where the leaderboard is, so I assumed there is no Leaderboard for Single Player games.

### CODE REVIEW

File: docs/wiki/Game-Compatibility.md [hunk 1]
```diff
@@ -67,3 +67,5 @@ While GOG does not officially support GOG Galaxy features (achievements and lead
...
 |[Quake II](https://www.gog.com/game/quake_ii_quad_damage)|5984 (11/01/2023)|🔲 Not available|[version `0.1.1`](https://github.com/imLinguin/comet/releases/tag/v0.1.1)|🔲 No|🟩 Achievements 🔲 Leaderboard|Achievements work as expected. The game needs OpenID support introduced in comet v0.1.1|
 |[Xeno Crisis](https://www.gog.com/game/xeno_crisis)|1.0.4 (2/11/2020)|❌ Not Working|[commit `55e4025`](https://github.com/imLinguin/comet/commit/55e402538df3bff354bf2e1e9a54fa4e5e091122)|🔲 No|🟩 Achievements 🔲 Leaderboard|Achievement connection does work. The GOG Galaxy communications are not present in the Linux version, thus the macOS or Windows version needs to be used with Comet to work.|
+|[Kingdom Come: Deliverance](https://www.gog.com/en/game/kingdom_come_deliverance)|1.9.6-404-504czi3 |🔲 Not available|[version `0.1.2`](https://github.com/imLinguin/comet/releases/tag/v0.1.2)|🟩 Yes|🟩 Achievements 🔲 Leaderboard||
+|[Tomb Raider GOTY](https://www.gog.com/en/game/tomb_raider_goty)|1.0|🔲 Not available|[version `0.1.2`](https://github.com/imLinguin/comet/releases/tag/v0.1.2)|🔲 No|🟩 Achievements 🔲 Leaderboard||
```
Comment (imLinguin) [inline docs/wiki/Game-Compatibility.md:71]:
```suggestion
|[Kingdom Come: Deliverance](https://www.gog.com/game/kingdom_come_deliverance)|1.9.6-404-504czi3 |🔲 Not available|[version `0.1.2`](https://github.com/imLinguin/comet/releases/tag/v0.1.2)|🟩 Yes|🟩 Achievements 🔲 Leaderboard||
|[Tomb Raider GOTY](https://www.gog.com/game/tomb_raider_goty)|1.0|🔲 Not available|[version `0.1.2`](https://github.com/imLinguin/comet/releases/tag/v0.1.2)|🔲 No|🟩 Achievements 🔲 Leaderboard||
```

**Review by imLinguin — CHANGES_REQUESTED** [review]
Thanks for the Pull request. I don't want to be picky, but maybe let's make the URLs not language specific (see the suggestion).

Also let's move those entries to be in alphabetical order with the rest.

**Comment by quantenzitrone** [comment]
i also fixed the alphabetical order of the rest if that is ok

**Review by imLinguin — APPROVED** [review]
Thanks!

**Pull request merged**